lib/smart_proxy_dynflow/plugin.rb

```diff
@@ -4,29 +4,18 @@
 
 class Proxy::Dynflow
   class Plugin < Proxy::Plugin
-    rackup_path = begin
-      require 'smart_proxy_dynflow_core'
-      'http_config_with_executor.ru'
-    rescue LoadError
-      'http_config.ru'
-    end
-    http_rackup_path File.expand_path(rackup_path, File.expand_path("../", __FILE__))
-    https_rackup_path File.expand_path(rackup_path, File.expand_path("../", __FILE__))
+    rackup_path = File.expand_path('http_config.ru', File.expand_path("../", __FILE__))
+    http_rackup_path rackup_path
+    https_rackup_path rackup_path
 
     settings_file "dynflow.yml"
     requires :foreman_proxy, ">= 1.12.0"
     default_settings :core_url => 'http://localhost:8008'
     plugin :dynflow, Proxy::Dynflow::VERSION
 
-    # rubocop:disable Lint/HandleExceptions
     after_activation do
-      begin
-        require 'smart_proxy_dynflow_core'
-      rescue LoadError
-        # Dynflow core is not available in the proxy, will be handled
-        # by standalone Dynflow core
-      end
+      # Ensure the core gem is loaded, if configure NOT to use the external core
+      require 'smart_proxy_dynflow_core' if Proxy::Dynflow::Plugin.settings.external_core == false
```

Comment:
```suggestion
      require 'smart_proxy_dynflow_core' unless Proxy::Dynflow::Plugin.settings.external_core
```

Reply:
The thing is I need to distinguish between `false` and `nil`. But maybe I'm just overthinking things and this whole three-valued logic is not needed at all

Reply:
Since there's duplication in the http_config, should we have a helper? Can we reuse the output of this loading somehow?

Reply:
Btw. would this work without Debian packaging/installer changes to explicitly disable the loading?

Reply:
Yes, by default the value will be unset. This will make the proxy try loading the core, use it if available and fallback to external core otherwise.